apisix/plugins/limit-count.lua

```diff
@@ -184,8 +185,10 @@ function _M.access(conf, ctx)
         return 500, {error_msg = "failed to limit count: " .. err}
     end
 
-    core.response.set_header("X-RateLimit-Limit", conf.count,
-                             "X-RateLimit-Remaining", remaining)
+    if conf.show_limit_quota_header == nil or conf.show_limit_quota_header then
```

Comment:
```suggestion
    if conf.show_limit_quota_header then
```

The default value will be set in `check_schema` even the configuration in etcd doesn't have it.

Reply:
okay, done~